components/codeblock/codeblock.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <pre>
+    <code
+      class="d-d-block d-bgc-secondary d-ws-pre-wrap d-p8 d-ba d-bc-subtle d-bar8 d-fs-200 d-lh-400 d-fc-secondary"
```

Comment:
Add `d-ff-mono` (SF Mono).

It currently renders browser default `monospace`, and Dialtone CSS doesn't apply it to `<code>` element (which it probably should, but that can ultimately be reconciled later in `dialtone` repo.